circuit/program/src/data/plaintext/to_bits.rs

```diff
@@ -49,6 +49,21 @@ impl<A: Aleo> ToBits for Plaintext<A> {
                 // Extend the vector with the bits of the struct.
                 vec.extend_from_slice(bits);
             }
+            Self::Array(elements, bits_le) => {
+                // Compute the bits of the array.
+                let bits = bits_le.get_or_init(|| {
+                    let mut bits_le = vec![Boolean::constant(true), Boolean::constant(false)]; // Variant bit.
+                    bits_le.extend(U32::constant(console::U32::new(elements.len() as u32)).to_bits_le());
```

Comment:
```suggestion
                    U32::constant(console::U32::new(elements.len() as u32)).write_bits_le(&mut bits_le));
```
this one doesn't allocate